src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -190,10 +200,34 @@ class EthService(
   }
 
   def submitHashRate(req: SubmitHashRateRequest): ServiceResponse[SubmitHashRateResponse] = {
-    //todo do we care about hash rate for now?
+    hashRate.updateAndGet(new UnaryOperator[Map[ByteString, (BigInt, Date)]] {
+      override def apply(t: Map[ByteString, (BigInt, Date)]): Map[ByteString, (BigInt, Date)] = {
+        val now = new Date
+        clearTimeouts(now, t + (req.id -> (req.hashRate, now)))
```

Comment:
ok but in documentation it says
`Hashrate, a hexadecimal string representation (32 bytes) of the hash rate`